config/rbac/rbac_role.yaml

```diff
@@ -89,7 +89,6 @@ rules:
   - cluster.k8s.io
   resources:
   - machines
-  - machines/status
```

Comment:
This change looks unintentional. Are we missing an rbac annotation for this?

Reply:
looks like I need another PR..